server/start_nginx.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "hi!"
+bash -c "/usr/sbin/nginx -g 'daemon off;'"
+echo "bye!"
```

Comment:
I can remove the echo's but the reason I've splitted those was because I wanted this script to run in the background.